metric/example_test.go

```diff
@@ -16,18 +16,23 @@ package metric_test
 
 import (
 	"context"
+	"database/sql"
 	"fmt"
+	"net/http"
 	"runtime"
 	"time"
 
 	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/attribute"
 	"go.opentelemetry.io/otel/metric"
+	semconv "go.opentelemetry.io/otel/semconv/v1.21.0"
 )
 
+var meter = otel.Meter("my-service-meter")
```

Comment:
While not reusing the meter every time makes a lot of sense, I don't know if suggesting to use a global one is a good idea, since it would be created before the MeterProvider is initialized and therefore always be a noop, and cause confusion.

Reply:
AFAIK after MeterProvider is initialized the returned meters are "switched" to ones that will be created by the new MeterProvider. See https://github.com/open-telemetry/opentelemetry-go/blob/4242228103c19cabc435a75b02d7aea82aa8bf36/internal/global/meter.go#L39-L44

Reply:
TIL!